Source/Microsoft.Teams.Apps.CompanyCommunicator/ClientApp/src/components/Messages/messages.tsx

```diff
@@ -161,6 +161,21 @@ class Messages extends React.Component<IMessageProps, IMessageState> {
     }
   }
 
+  private sendingNumbers = (message: any) => {
```

Comment:
Let's change this name to renderSendingNumbersText()

Reply:
done